spec/support/features/doc_auth_helper.rb

```diff
@@ -223,10 +223,13 @@ def mock_doc_auth_acuant_error_unknown
     )
   end
 
-  def set_up_document_capture_result(uuid:, idv_result:, pii:)
+  def set_up_document_capture_result(uuid:, idv_result:)
     dcs = DocumentCaptureSession.where(uuid: uuid).first_or_create
-    dcs.store_proofing_pii_from_doc(pii) # generates a result_id
```

Comment:
do we need to set a `result_id`? Since the `#store_doc_auth_result` won't make one?

Reply:
`create_doc_auth_session`creates it 🙂